content/english/history.html

```diff
@@ -88,8 +88,8 @@ <h3><time>December '20</time></h3>
 {{< /history-entry >}}
 
 {{< history-entry graph="void-left point-left red">}}
-    <h3><time>Coming 2021</time></h3>
-    The community will release HedgeDoc 2.0.
+    <h3><time>Coming (hopefully) 2021</time></h3>
```

Comment:
```suggestion
    <h3><time>2021</time></h3>
    Tentative release date of HedgeDoc 2
```
(sorry, GitHub seemingly can't do multiline suggestions)

What about this?
It should make clear that the 2021 date is not guaranteed, which should be all we want to achieve here?

Reply:
I'm okay with that

Reply:
I changed it.